prerequisites.rst

```diff
@@ -1,35 +1,250 @@
-Prerequisites
+Requisites
 =============
 
-Before installing |omv| make sure your hardware is supported.
+The |omv| system is the most flexible Network Attached Storage (NAS) on the market,
+this is because it relies on the operating system Debian GNU/Linux [9]_
 
-* **CPU**: Any x86-64 or ARM compatible processor
-* **RAM**: 1 GiB capacity
-* **HDD**:
+Any system has software requirements and hardware requirements, also minimal
```

Comment:
```suggestion
Any system has software and hardware requirements, also minimal
```

Reply:
fixed in commit be624bd8da52ad54039c7533a818c8b7073b30b2